experimental/packages/otlp-exporter-base/src/platform/browser/util.ts

```diff
@@ -101,3 +101,57 @@ export function sendWithXhr(
     }
   };
 }
+
+/**
+ * function to send metrics/spans using browser fetch
+ *     used when navigator.sendBeacon and XMLHttpRequest are not available
+ * @param body
+ * @param url
+ * @param headers
+ * @param onSuccess
+ * @param onError
+ */
+export function sendWithFetch(
+  body: string,
+  url: string,
+  headers: Record<string, string>,
+  exporterTimeout: number,
+  onSuccess: () => void,
+  onError: (error: OTLPExporterError) => void
+): void {
+  const controller = new AbortController();
+  const timerId = setTimeout(() => controller.abort(), exporterTimeout);
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      ...defaultHeaders,
+      ...headers,
+    },
+    signal: controller.signal,
+    body,
+  })
+    .then(response => {
+      if (response.ok) {
+        response.text().then(
+          t => diag.debug('Request Success', t),
+          () => {}
+        );
+        onSuccess();
+      } else {
+        onError(
+          new OTLPExporterError(
+            `Failed to export with fetch: (${response.status} ${response.statusText})`,
+            response.status
+          )
+        );
+      }
+    })
+    .catch((e: Error) => {
```

Comment:
This catch block may also catch errors from the `then` handler. I think possibly you meant to only catch errors from the `fetch` itself? If so, I would move this error handler to be the second argument of `then`